nsqd/nsqd.go

```diff
@@ -504,6 +545,13 @@ func (n *NSQD) Notify(v interface{}) {
 			n.Unlock()
 		}
 	})
+
+	n.waitGroup.Wrap(func() {
+		select {
+		case <-n.exitChan:
+		case n.gossipChan <- v:
```

Comment:
still think it needs documenting? (I don't, given it's context within the `Notify` func)